tests/util/mod.rs

```diff
@@ -31,20 +28,23 @@ pub fn jvm() -> &'static Arc<JavaVM> {
     unsafe { JVM.as_ref().unwrap() }
 }
 
+pub fn attach_current_jvm_thread() -> AttachGuard<'static> {
```

Comment:
`attach_current_thread`? Or something more verbose?

You attach a _native_ thread to the JVM, not a _JVM_ thread to anything.